src/sphinx_codelinks/sphinx_extension/directives/src_trace.py

```diff
@@ -109,6 +148,9 @@ def run(self) -> list[nodes.Node]:
         target_dir = out_dir / src_dir.name
 
         extra_options = {"project": project}
```

Comment:
Name confusing:
https://sphinx-needs.readthedocs.io/en/latest/configuration.html#needs-extra-options

Reply:
changed to `additional_options`